src/menu/example/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Don't need the specific widget example index.html anymore, this is centralised in `common`